components/treelist/templates/column-chooser.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Column Chooser
+page_title: TreeList - Column Chooser Template
+description: Use the Column Chooser Template of the TreeList to control the rendering and customize the content of the Column Chooser.
+slug: treelist-templates-column-chooser
+tags: telerik,blazor,treelist,templates,column,menu
+published: True
+position: 40
+---
+
+# Column Chooser Template
+
+The Column Chooser Template lets you control the rendering of the [ColumnChooser]({%slug treelist-column-menu%}#column-chooser). It exposes a `context` object that contains a list of all columns in the TreeList. 
```

Comment:
What is the type of the `context` object? Link to the API reference (let me know if you need help with linking an API page that doesn't exist yet).